src/elements/content-open-with/__tests__/ContentOpenWith.test.js

```diff
@@ -299,7 +299,7 @@ describe('elements/content-open-with/ContentOpenWith', () => {
 
             displayIntegration = {
                 appIntegrationId: '1',
-                displayName: 'Adobe Sign',
+                displayName: 'Google Docs',
```

Comment:
It looks like this file has a few more mentions of "adobe" that could be removed

Reply:
Adobe is still supported because I see the ability to integrate with adobe acrobat, so that's why I left it in. Should I remove it?

Reply:
Ah gotcha, I'd been assuming that only Adobe Sign is associated with Content Open With, and that ADOBE_INTEGRATION_ID was referring to Adobe Sign. It would be good to confirm with @box/partners whether "Adobe" here is referring to adobe acrobat or adobe sign.

Reply:
I don't think we ever added adobe acrobat to Content Open With, but it is available in the webapp. It looks like all the usages of Adobe in this file are referencing mock data, and using fake IDs. You could remove it, but then we'd lose test cases for when there's more than 2 integrations. It might be better to rename it to a fake app name to retain the test coverage for when there's > 1 integration